playbooks/pre-common.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Install Bats' support library
+  git:
```

Comment:
Actually, we don't. I added this before to unify the `setup-env.yaml`, to avoid the duplication of this. But if we unify the `setup-env.yml` we could move this code into that file.

Reply:
Ok, understood.